macros/shared

```diff
@@ -225,6 +263,23 @@ CROSS_COMPILATION_CONF_EOF\
 %cross_scan_attribution \
   bottlerocket-license-scan --spdx-data /usr/libexec/tools/spdx-data --out-dir %{buildroot}%{_cross_attribution_vendor_dir}
 
+%set_cross_cmake_env \
+  %write_cross_cmake_toolchain_conf \
+  CMAKE_TOOLCHAIN_FILE="${PWD}/%{_cross_cmake_toolchain_conf}" ; \
+  export CMAKE_TOOLCHAIN_FILE ; \
+  %{nil}
+
+%set_cross_cmake_env_static \
+  %write_cross_cmake_toolchain_conf_static \
+  CMAKE_TOOLCHAIN_FILE="${PWD}/%{_cross_cmake_toolchain_conf_static}" ; \
+  export CMAKE_TOOLCHAIN_FILE ; \
+  %{nil}
+
+%set_cross_bindgen_env \
+  BINDGEN_EXTRA_CLANG_ARGS="--target=%{_cross_target} --sysroot=%{_cross_rootdir}" ; \
+  export BINDGEN_EXTRA_CLANG_ARGS ; \
+  %{nil}
```

Comment:
The need for these goes away if you just move them into the respective `%_cargo` environment macros. But you're missing a `%set_cross_bindgen_env_static` if you decide to keep them.